.travis.yml

```diff
@@ -1,14 +1,27 @@
-# GCE VMs have better performance (will be upgrading to premium VMs soon)
-sudo: required
+dist: xenial
+group: stable
 
 language: scala
-jdk: openjdk8
 
 cache:
   directories:
     - $HOME/.ivy2/cache
     - $HOME/.sbt
     - $HOME/.rvm/
+    - $HOME/.sdkman
+
+before_install:
+  # adding $HOME/.sdkman to cache would create an empty directory, which interferes with the initial installation
+  - "[[ -d /home/travis/.sdkman/ ]] && [[ -d /home/travis/.sdkman/bin/ ]] || rm -rf /home/travis/.sdkman/"
```

Comment:
Should we stick to `$HOME/.sdkman` seeing as those that's what's used in `cache.directory`?

(Also you can drop the `[[ -d /home/travis/.sdkman/ ]]` check and just check for `[[ -d "$HOME/.sdkman/bin/" ]]`.)

Reply:
I sort of remember `$HOME` not always working in some parts of the travis yaml. Maybe due to the initialization ordering these scripts are executed? I don't have specific memory on this being one of those situations, or if I was just being paranoid.

Reply:
Happy to change these to `$HOME`?

Reply:
Yea. I am happy to give it a shot again.

Reply:
5f04df40618f9213e36be38d687a7ba550007afd